content/04-postgresql/07-create-and-delete-databases-and-tables.mdx

```diff
@@ -561,3 +561,63 @@ This will remove the database and any child elements, like tables, within.
 This article covered the basics of how to create and delete databases and tables within PostgreSQL.  These are some of the most basic commands required to set up a database system and being defining the structure of your data.
 
 As mentioned earlier, the SQL statements covered in this PostgreSQL tutorial, particularly the `CREATE TABLE` statement, have many additional parameters can be used to change PostgreSQL's behavior.  You can find out more about these by checking out the [official PostgreSQL documentation](https://www.postgresql.org/docs/current/sql-createtable.html).
+
+## FAQ
+
+
+<details><summary>Does PostgreSQL support the `IF NOT EXISTS` clause when using the `CREATE DATABASE` command?</summary>
+
+Yes, PostgreSQL supports the use of [`IF NOT EXISTS`](https://www.prisma.io/dataguide/postgresql/create-and-delete-databases-and-tables#create-tables-conditionally-with-the-if-not-exists-clause) when creating both databases and tables. The below demonstrates using the clause for table creation. 
+
+```sql
+CREATE TABLE IF NOT EXISTS table_name (
+    column_name TYPE [column_constraint],
+    [table_constraint,]
+);
+```
+
+</details>
+
+<details><summary>How do you create a database from a dump in PostgreSQL?</summary>
+
+To create a database from a dump ([pg_dump](https://www.prisma.io/dataguide/mongodb/mongodb-datatypes#date)), PostgreSQL provides the [utility program `pg_restore`](https://www.postgresql.org/docs/current/app-pgrestore.html).
+
+This program recreates the database in the same state as it was at the time of the dump. Example syntax would look like the following:
+
+```sql
+pg_restore [connection-option...][option...][filename]
+```
+
+</details>
+
+<details><summary>Which command line creates a database in PostrgreSQL?</summary>
+
+To create a database in PostgreSQL, use the [`createdb` command](https://www.postgresql.org/docs/current/app-createdb.html). The syntax is as follows:
+
+```sql
```

Comment:
The command here is actually executed in the shell instead of from the PostgreSQL client.  So the ```sql is probably not a fit and the ending semi-colon may throw off the interpretation of the command.